liiweb/templates/liiweb/legislation_list.html

```diff
@@ -6,8 +6,10 @@
 {% block page-content %}
   {% block breadcrumbs %}{% endblock %}
   {% block entity-profile %}
-    {% if entity_profile %}
-      <div class="mt-3">{% include 'peachjam/_entity_profile.html' %}</div>
+    {% if jurisdiction_entity_profile %}
```

Comment:
You don't need to make these changes -- just ensure the context variables `entity_profile` and `entity_profile_title` are in place. The templates will do everything else.